assets/js/blocks/cart/style.scss

```diff
@@ -21,7 +21,7 @@ table.wc-block-cart-items td {
 	background: none !important;
 	// Remove borders on default themes.
 	border: 0;
-	margin: 0;
+	margin: 0 0 2em;
```

Comment:
This margin will be applied regardless of the cross-sells block's presence, right? Are we OK with this?